### PR TITLE
Setting session properties

### PR DESCRIPTION
**Pull request by hovaesco**
Addresses: https://github.com/findinpath/dbt-trino/issues/4

### CODE REVIEW

File: test/unit/test_adapter.py [hunk 1]
```diff
@@ -25,6 +25,7 @@ def setUp(self):
                     'schema': 'dbt_test_schema',
                     'method': 'none',
                     'user': 'trino_user',
+                    'session_properties': {"query_max_run_time": "5d"}
```
Comment (hovaesco) [inline test/unit/test_adapter.py:28]: I am not really sure whether that test is actually relevant.

Comment (findinpath) [reply test/unit/test_adapter.py:28]: Please do specify multiple session properties in the dictionary.


Can't you check the contents of the session properties dictionary in the test?

File: README.md [hunk 2]
```diff
@@ -39,6 +39,7 @@ A dbt profile can be configured to run against Trino using the following configu
 | password  | Password for authentication | Optional (required if `method` is `ldap` or `kerberos`)  | `none` or `abc123` |
 | http_headers | HTTP Headers to send alongside requests to Trino, specified as a yaml dictionary of (header, value) pairs. | Optional |  |
 | http_scheme | The HTTP scheme to use for requests to Trino | Optional (default is `http`, or `https` for `method: kerberos` and `method: ldap`) | `https` or `http`
+| session_properties | Sets Trino session properties used in the connection | Optional | `{"query_max_run_time": "5d"}`
```
Comment (findinpath) [inline README.md:42]: Please specify multiple properties here to give a clue to the user what the separator for multiple properties is.

Comment (findinpath) [reply README.md:42]: How do we set an empty value for a session property in the dictionary? `query_max_scan_physical_bytes`  has no value set when executing `SHOW SESSION`

Comment (hovaesco) [reply README.md:42]: We cannot set empty value for `query_max_scan_physical_bytes` at all. 
```
trino> set session query_max_scan_physical_bytes= '';
Query 20210922_091533_03496_ieee5 failed: line 1:1: size is empty
```

File: test/unit/test_adapter.py (hunk 1 above)
Comment (hovaesco) [reply test/unit/test_adapter.py:28]: I meant that if I add random key-value pair test still pass.

**Comment by findinpath** [comment]
Thank you @hovaesco for your valuable contribution.

**Pull request merged**